src/ValuedTextInput/valuedTextInput.css

```diff
@@ -102,19 +110,48 @@
     }
 }
 
+
 .fakeHovered
 {
+    .container
+    {
+        outline:        0.1rem solid var( --PC-BLUE );
+    }
+
+    .input
+    {
+        outline:        none;
+    }
+
     .valueLabel
     {
         border-color:   var( --PC-BLUE );
     }
 }
 
+
 .error
 {
+    .container
+    {
+        outline:        0.1rem solid var( --PC-RED );
+    }
+
+    .input
+    {
+        outline:        none !important;
```

Comment:
Since `input` never has an outline this should be set somewhere above, not here.

Also instead of using `!important` we can do `.input:focus { outline: none; }`.